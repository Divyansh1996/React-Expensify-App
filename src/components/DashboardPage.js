import React from "react";

import ExpenseList from "./ExpenseList";
import ExpenseFilterInput from "./ExpenseFilterInput";

const DashboardPage = () => (
    <div>
        <ExpenseFilterInput />
        <ExpenseList />
    </div>
)


export default DashboardPage;