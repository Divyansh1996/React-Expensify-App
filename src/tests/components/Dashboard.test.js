import React from "react";
import DashboardPage from "../../components/DashboardPage";
import { shallow } from "enzyme";

test("Testing the Dashboard page", () => {
    const wrapper = shallow(<DashboardPage/>)
    expect(wrapper).toMatchSnapshot()
})