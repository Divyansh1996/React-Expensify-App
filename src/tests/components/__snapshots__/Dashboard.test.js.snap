// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Testing the Dashboard page 1`] = `
<div>
  <Connect(ExpenseFilterInput) />
  <Connect(ExpenseList) />
</div>
`;
